cornice_swagger/converters/parameters.py

```diff
@@ -36,6 +36,30 @@ def convert(self, schema_node, definition_handler):
 class PathParameterConverter(ParameterConverter):
     _in = 'path'
 
+    def convert(self, schema_node, definition_handler):
+        converted = super(PathParameterConverter, self).convert(schema_node,
+                                                                definition_handler)
+        name = converted['name']
+        pattern = None
+
+        # Extract regex pattern from name
+        template = name.split(':', 1)
+        name = template[0]
+        if len(template) == 2:
+            pattern = template[1]
+
+        # handle traverse and subpath
+        # docs.pylonsproject.org/projects/pyramid/en/latest/narr/hybrid.html
+        for subpath_marker in ('*subpath', '*traverse'):
+            if name.find(subpath_marker) > 0:
+                name = subpath_marker[1:]
```

Comment:
I'm not sure about this one, but I think we can just ignore the pattern in such cases.